src/Model.jl

```diff
@@ -58,13 +58,17 @@ struct Model{T <: Real, VT <: Real}
 
     # list of symmetries of the model
     symmetries::Vector{SymOp{VT}}
+
+    # Allows to fix the Fermi level to a certain value. By default (value of `nothing`), the
+    # level depends on the number of electrons.
```

Comment:
Mention that this makes DFTK do simulations in the grand-canonical ensemble instead of the canonical ensemble.